src/lib/components/inbox/conversations/conversation_snippet.tsx

```diff
@@ -113,31 +115,37 @@ export class ConversationSnippet extends React.Component<Props, any> {
     const imageURL = artwork.image.url
 
     return (
-      <Card>
-        <CardContent>
-          <OpaqueImageView imageURL={imageURL} style={styles.image} />
-          <TextPreview>
-            <HorizontalLayout>
-              <SmallHeadline>{galleryName}</SmallHeadline>
-              <DateHeading>
-                <MetadataText>{date}</MetadataText>
-                <UnreadIndicator />
-              </DateHeading>
-            </HorizontalLayout>
-            <HorizontalLayout>
-              <P>
-                <ArtworkSubtitle>{artworkArtist}</ArtworkSubtitle>
-                <ArtworkTitle>{artworkTitle}</ArtworkTitle>
-                <ArtworkSubtitle>{artworkDate}</ArtworkSubtitle>
-              </P>
-            </HorizontalLayout>
-            <P>{conversationText}</P>
-          </TextPreview>
-        </CardContent>
-        <Separator />
-      </Card>
+      <TouchableWithoutFeedback onPress={this.handleTap.bind(this)}>
+        <Card>
+          <CardContent>
+            <OpaqueImageView imageURL={imageURL} style={styles.image} />
+            <TextPreview>
+              <HorizontalLayout>
+                <SmallHeadline>{galleryName}</SmallHeadline>
+                <DateHeading>
+                  <MetadataText>{date}</MetadataText>
+                  <UnreadIndicator />
+                </DateHeading>
+              </HorizontalLayout>
+              <HorizontalLayout>
+                <P>
+                  <ArtworkSubtitle>{artworkArtist}</ArtworkSubtitle>
+                  <ArtworkTitle>{artworkTitle}</ArtworkTitle>
+                  <ArtworkSubtitle>{artworkDate}</ArtworkSubtitle>
+                </P>
+              </HorizontalLayout>
+              <P>{conversationText}</P>
+            </TextPreview>
+          </CardContent>
+          <Separator />
+        </Card>
+      </TouchableWithoutFeedback>
     )
   }
+
+  handleTap() {
+    SwitchBoard.presentNavigationViewController(this, "/conversation/")
```

Comment:
I think it would make more sense to have the routing navigation controller in the conversations/parent component and expose on `onSelected` callback from the ConversationSnippet component instead.
That way we don't couple routing with that child component

```typescript
interface Props {
  onSelected: () => void
}
```